sstmac/hardware/topology/topology.cc

```diff
@@ -88,18 +88,24 @@ int Topology::nproc = 0;
 SwitchId
 Topology::nodeToLogpSwitch(NodeId nid) const
 {
-  int n_nodes = numNodes();
-  int nodes_per_switch = n_nodes / nproc;
-  int epPlusOne = nodes_per_switch + 1;
-  int num_procs_with_extra_node = n_nodes % nproc;
-
-  int div_cutoff = num_procs_with_extra_node * epPlusOne;
-  if (nid >= div_cutoff){
-    int offset = nid - div_cutoff;
-    return offset / nodes_per_switch;
-  } else {
-    return nid / epPlusOne;
-  }
+  // currently using single switch topology for short circuit network when using sst-core
+  // and I think the code below is wrong for some corner cases anyways
+
+  spkt_abort_printf("unexpected call to nodeToLogPSwitch()");
+  return 0;
+
+//  int n_nodes = numNodes();
```

Comment:
If we want to go back to one LogP switch per node for sst-core (for performance reasons) it'd be way easier to have this code here to start with, rather than having to dig through history to figure out where it got deleted. IMO

Reply:
Can you just put a comment on why you didn't delete then?

Reply:
Although it's not like it wouldn't be in history so it's not actually lost if you delete it.

Reply:
Added comment in cs 5841f71.